enterprise/server/usage/usage.go

```diff
@@ -101,16 +99,25 @@ type tracker struct {
 	stopFlush chan struct{}
 }
 
-func NewTracker(env environment.Env, clock timeutil.Clock, flushLock interfaces.DistributedLock, rbuf *redisutil.CommandBuffer, opts *TrackerOpts) *tracker {
+func NewTracker(env environment.Env, clock timeutil.Clock, flushLock interfaces.DistributedLock, rbuf *redisutil.CommandBuffer) (*tracker, error) {
+	if !usage_config.UsageTrackingEnabled() {
```

Comment:
nit: it feels a bit unconventional for a `New*` func to return `nil, nil` -- would it work to move this enabled check to the callsite instead?